platform/kobo/koreader.sh

```diff
@@ -230,7 +230,7 @@ ko_do_fbdepth() {
 # Fun fact: this wouldn't be necessary if Kobo were using a non-prehistoric glibc... (it was fixed in glibc 2.26).
 ko_do_dns() {
     # If there aren't any servers listed, append CloudFlare's
-    if not grep -q '^nameserver' "/etc/resolv.conf"; then
```

Comment:
It's perfectly valid syntax after all. You're running the program `not` with some arguments. ;-P

Reply:
Indeed ^^.

And either ZSH has a builtin for it, or I checked that locally, where LLVM does ship one which probably do the obvious ^^.